### DISCUSSION TITLE
Next release (0.9.3)

### DISCUSSION THREAD

**Discussion by EwoutH** [gephi/gephi#2353] [Q&A] [answered]
Currently gephi 0.9.2 is over four years old. I would love to use a stable version of gephi with all the accumulated [improvements](https://github.com/gephi/gephi/compare/v0.9.2...master). I noticed there is a [0.9.3 milestone](https://github.com/gephi/gephi/milestone/24), but no due date.

Of course a pre-release or release candidate before the stable release could also be useful.

**Comment by mbastian** [comment, marked as answer]
Hi there, 

We have [a developer retreat](https://gephi.wordpress.com/2021/08/23/call-for-participants-gephi-code-sustainability-retreat-2021/) planned at the end of this month. We hope to make good progress over there. If this is the case, I think we should be able to release a new version by the end of this year.

Nightly builds are normally available (see links on README) but at the moment builds are paused because we've exhausted our free credits on Travis-CI. Once this is resolved, one could download the 0.9.3 snapshots as we make progress. That said, the current version already includes some patches so feel free to use it instead of the 0.9.2.

Also, let me move this to discussion - thanks!

**Comment by EwoutH** [comment]
Thanks for the quick reply! Looking forward to it.

Maybe related, I have created a simple GitHub Actions CI configuration in #2354. Currently it builds and tests Gephi on Ubuntu, Windows and macOS, and could potentially be extended to also publish packages. Creating a separate workflow to run nightlies should also be possible.

**Reply by mbastian** [reply]
Hi and thanks for the PR! Moving to GitHub Actions is indeed the plan and I'm working on it. Hopefully should be done in a couple of days, ahead of the retreat.
